src/platform/user/authorization/components/RequiredLoginView.jsx

```diff
@@ -134,6 +143,10 @@ export const RequiredLoginView = props => {
     });
   };
   const renderWrappedContent = () => {
+    if (user.login.error) {
```

Comment:
We should probably put this under the `user` store so that it'll be `user.login`, `user.profile`, and `user.errors`